test/components/CalendarDay_spec.jsx

```diff
@@ -84,6 +92,10 @@ describe('CalendarDay', () => {
       handleDayMouseUpSpy = sinon.spy(CalendarDay.prototype, 'handleDayMouseUp');
     });
 
+    afterEach(() => {
+      sinon.restore();
```

Comment:
https://github.com/airbnb/react-dates/blob/master/package.json#L15 needs to be updated to explicitly pass the _helpers paths to mocha first, before the rest of the tests.